app/js/rooms.js

```diff
@@ -21,7 +21,8 @@ require(
             } else {
```

Comment:
lines 8, 9, 15, 65 there's a hanging comma which apparently can cause issues with javascript engine implementations (according to eslint)